linkage/demo_notebooks/2022_10_21a_prl_splink_test_part_1.ipynb

```diff
@@ -0,0 +1,3242 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    df1.reindex(df2.index).compare(df2)
</pre><p>Is it expected that all the names are NaN here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research_prl/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>